docs/content/porter-or-duffle.md

```diff
@@ -1,59 +1,95 @@
 ---
-title: Porter vs. Duffle
-description: Does Porter Replace Duffle?
+title: Porter or Duffle?
+description: A comparison of Porter, Duffle and when you would choose one over the other
 ---
 
-<p align="center">NOPE! 😁</p>
+<h5>A comparison of porter, duffle and when you would choose one over the other</h5>
```

Comment:
nit: Should we capitalize Porter and Duffle here, for consistency?

Reply:
Yes, thanks! I'll fix that before I merge